src/js/components/devices/expanded-device.js

```diff
@@ -174,7 +174,7 @@ export class ExpandedDevice extends React.Component {
           <div className="device-connect bordered report-list">
             <h4 className="margin-bottom-small">Remote Terminal</h4>
             <div className="flexbox" style={{ flexDirection: 'row' }}>
-              {connect_status === DEVICE_CONNECT_STATES.unknown && (
+              {(connect_status === undefined || connect_status === DEVICE_CONNECT_STATES.unknown) && (
```

Comment:
you could set it like we do for the [device auth status](https://github.com/mendersoftware/gui/blob/master/src/js/components/devices/expanded-device.js#L97) but for the single check here the undefined condition will do just as good... though instead of masking it, a migration of existing devices in the backend would be my first choice 😉

Reply:
@mzedel does it look good, now? https://github.com/mendersoftware/gui/pull/1249/commits/efcb99cda2cc61309bef466211c1366efa03e737